airbyte-integrations/connectors/source-github/metadata.yaml

```diff
@@ -6,6 +6,7 @@ data:
   connectorType: source
   definitionId: ef69ef6e-aa7f-4af1-a01d-ef775033524e
   dockerImageTag: 0.4.8
+  maxSecondsBetweenMessages: 5400
```

Comment:
How did we decide on these values?

Reply:
Good question @girarda !

So the we're currently migrating to use these metadata files. 

But they are not in use yet. 

The changes you see are all related to ensuring our new oss_registry.json files match the current oss_catalog.json files that are currently in use by the airbyte system. 

So these values are all just taken from the already deployed catalog file. 

You can see these in the code base in the _definitions.yaml files as well. 

I can share some links Monday when I'm near my laptop 🙂